python/glow/gwas/lin_reg.py

```diff
@@ -211,3 +210,32 @@ def _linear_regression_inner(genotype_pdf: pd.DataFrame, Y_state: YState,
     out_df['phenotype'] = phenotype_names.repeat(num_genotypes).tolist()
 
     return out_df
+
+
+def _generate_linreg_output(genotype_df, sql_type, Y_state, Y_mask, Y_scale, Q, dof, phenotype_df,
+                            Y_raw_nan_filled, verbose_output) -> DataFrame:
+    # Construct output schema
+    result_fields = [
+        StructField('effect', sql_type),
+        StructField('stderror', sql_type),
+        StructField('tvalue', sql_type),
+        StructField('pvalue', sql_type),
+        StructField('phenotype', StringType())
+    ]
+
+    if verbose_output:
+        result_fields += ([
+            StructField('n', IntegerType()),
+            StructField('sum_x', sql_type),
+            StructField('y_transpose_x', sql_type)
+        ])
+
+    result_struct = gwas_fx._output_schema(genotype_df.schema.fields, result_fields)
+
+    def map_func(pdf_iterator):
+        for pdf in pdf_iterator:
+            yield gwas_fx._loco_dispatch(
+                pdf, Y_state, _linear_regression_inner, Y_mask, Y_scale, Q, dof,
+                phenotype_df.columns.to_series().astype('str'), Y_raw_nan_filled, verbose_output)
```

Comment:
Is there a reason two different names are used in the code for this? `Y_raw_nan_filled` and `Y_for_verbose_output`?

Reply:
No, we can use Y_for_verbose_output throughout